src/TLuaInterpreter.cpp

```diff
@@ -111,61 +111,66 @@ static const char *bad_cmdline_value = "command line \"%s\" not found";
+        if (!lua_isstring(_L, pos_)) {                                                         \
+            lua_pushfstring(_L, bad_cmdline_type, __FUNCTION__, pos_, luaL_typename(_L, pos_));\
             return lua_error(_L);                                                              \
         }                                                                                      \
-        lua_tostring(_L, pos);                                                                 \
+        lua_tostring(_L, pos_);                                                                \
     })
     
 #define CONSOLE_NIL(_L, _name)                                                                 \
     ({                                                                                         \
-        auto name = (_name);                                                                   \
-        auto console = getHostFromLua(_L).findConsole(name);                                   \
-        console;                                                                               \
+        auto name_ = (_name);                                                                  \
+        auto console_ = getHostFromLua(_L).findConsole(name_);                                 \
+        console_;                                                                              \
     })
 
 #define CONSOLE(_L, _name)                                                                     \
     ({                                                                                         \
-        auto name = (_name);                                                                   \
-        auto console = getHostFromLua(_L).findConsole(name);                                   \
-        if (!console) {                                                                        \
+        auto name_ = (_name);                                                                  \
+        auto console_ = getHostFromLua(_L).findConsole(name_);                                 \
+        if (!console_) {                                                                       \
             lua_pushnil(L);                                                                    \
-            lua_pushfstring(L, bad_window_value, name.toUtf8().constData());                   \
+            lua_pushfstring(L, bad_window_value, name_.toUtf8().constData());                  \
             return 2;                                                                          \
         }                                                                                      \
-        console;                                                                               \
+        console_;                                                                              \
     })
 
 #define COMMANDLINE(_L, _name)                                                                 \
```

Comment:
This strikes me as worrying - as I understand it identifiers with a leading underscore followed by an UPPER-CASE letter is reserved for the compiler and may NOT be used - https://stackoverflow.com/a/1449301/4805858

Reply:
This is a macro parameter, not an "identifier". Whatever the compiler has assigned to this name, if anything, will be masked within the macro. It'll be expanded by the preprocessor and thus isn't visible to the compiler itself, let alone the linker.